### PR TITLE
sci-calculators/gonvert: Version bump to 0.2.39

### PR DESCRIPTION
**Pull request by g-braeunlich**
Package-Manager: portage-2.3.0

### CODE REVIEW

File: sci-calculators/gonvert/gonvert-0.2.39.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+DEPEND="
+	${PYTHON_DEPS}
+	dev-python/pygtk:2[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
+
+PATCHES=( "${FILESDIR}"/0.2.23-paths.patch )
+
+src_install () {
+	emake install DESTDIR="${D}" prefix="${EPREFIX}/usr"
+	python_fix_shebang "${ED}"/usr/bin
```
Comment (SoapGentoo) [inline sci-calculators/gonvert/gonvert-0.2.39.ebuild]: `"${ED%/}"` (double-slashing, remember?)

File: sci-calculators/gonvert/gonvert-0.2.39.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	rm -fr "${ED}/usr/share/doc/${PN}"
```
Comment (SoapGentoo) [inline sci-calculators/gonvert/gonvert-0.2.39.ebuild]: `"${ED%/}"`

File: sci-calculators/gonvert/gonvert-0.2.39.ebuild [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	${PYTHON_DEPS}
+	dev-python/pygtk:2[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
```
Comment (SoapGentoo) [inline sci-calculators/gonvert/gonvert-0.2.39.ebuild]: swap `RDEPEND` and `DEPEND` again

File: sci-calculators/gonvert/gonvert-0.2.39.ebuild (hunk 1 above)
Comment (g-braeunlich) [inline sci-calculators/gonvert/gonvert-0.2.39.ebuild]: Sorry!

File: sci-calculators/gonvert/gonvert-0.2.39.ebuild [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	${PYTHON_DEPS}
+	dev-python/pygtk:2[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}"
+
+PATCHES=( "${FILESDIR}"/0.2.23-paths.patch )
+
+src_install () {
+	emake install DESTDIR="${D}" prefix="${EPREFIX}/usr"
+	python_fix_shebang "${ED%/}"/usr/bin
+	rm -fr "${ED%/}/usr/share/doc/${PN}"
```
Comment (SoapGentoo) [inline sci-calculators/gonvert/gonvert-0.2.39.ebuild]: sorry forgot to say: missing `|| die` (even on `-f` !)

**Pull request closed without merging**